simpledotorg/simple-server
# Add graph to show the diabetes visits breakdown

**Story card:** [sc-7783](https://app.shortcut.com/simpledotorg/story/7783/add-diabetes-treatment-outcomes-graph)

## Because

We need to add a graph showing the breakdown of diabetic visits

## This addresses

Adds HTML and javascript changes necessary to show the breakdown graph

## Test instructions

1. Turn on diabetes_management_reports on the flipper
2. Visit the reports page of a facility which has enable_diabetes_management set to true
3. Click on the Diabetes link
4. You should see the Treatment outcomes of diabetes patients under care graph

<img width="1344" alt="Screenshot 2022-05-10 at 8 37 26 PM" src="https://user-images.githubusercontent.com/32141642/167661247-3b542dbb-aac4-4115-9b41-ef39cf68192f.png">
